common/src/main/java/net/neoforged/gradle/common/extensions/subsystems/SubsystemsExtension.java

```diff
@@ -33,13 +29,27 @@ public SubsystemsExtension(Project project) {
         configureRecompilerDefaults();
         configureParchmentDefaults();
         configureToolsDefaults();
+        configureDevLoginDefaults();
+    }
+
+    private void configureDevLoginDefaults() {
+        DevLogin devLogin = getDevLogin();
+        devLogin.getEnabled().convention(
+                getBooleanProperty("devLogin.enabled").orElse(true)
```

Comment:
Wasn't it meant to be false by default?

Reply:
By lukes suggestion, and the one I agreed with, the subsystem it self is enabled, yet the default on each run is `false` so the user only needs to enable the relevant runs he or she wants.